src/modules/jcms_rest/src/Plugin/rest/resource/LabsExperimentsItemRestResource.php

```diff
@@ -60,7 +60,7 @@ public function get(int $number) {
       return $response;
     }
 
-    throw new NotFoundHttpException(t('Lab experiment with ID @id was not found', ['@id' => $number]));
+    throw new JcmsNotFoundHttpException(t('Lab experiment with ID @id was not found', ['@id' => $number]), NULL, 'application/problem+json');
```

Comment:
A minor thing but if this is the preferred convention `JcmsNotFoundHttpException` rather than `JCMSNotFoundHttpException` then we need to change the other class and file names that are prefixed with `JCMS`.